common/util/webhook_utils.go

```diff
@@ -0,0 +1,60 @@
+package util
+
+import (
+	"errors"
+	"fmt"
+	"net/url"
+	"os"
+	"strconv"
+	"strings"
+
+	"github.com/google/uuid"
+	"k8s.io/apimachinery/pkg/util/validation/field"
+)
+
+func ValidateUUID(u string) error {
+	_, err := uuid.Parse(u)
+	return err
+}
+
+func ValidateURL(u string) error {
+	_, err := url.ParseRequestURI(u)
+	if err != nil {
+		return err
+	}
+
+	return nil
+}
+
+func CombineFieldErrors(fieldErrors field.ErrorList) error {
+
+	if len(fieldErrors) == 0 {
+		return nil
+	}
+
+	var errorStrings []string
+	for _, fe := range fieldErrors {
+		errorStrings = append(errorStrings, fe.Error())
+	}
+	return errors.New(strings.Join(errorStrings, "; "))
+}
```

Comment:
I believe it can be used to combine multiple errors when you have individual errors as shown in the example:

	err1 := errors.New("err1")
	err2 := errors.New("err2")
	err := errors.Join(err1, err2)
	
	
We have an array of errors here which we are trying to append together, which means we will need to loop through the allErrs anyway, basically voiding the intention to use Join().

Reply:
I am closing this comment if it answers your question, feel free to reopen if you have further questions.